app/controllers/metrics_controller.rb

```diff
@@ -2,4 +2,14 @@ class MetricsController < ApplicationController
 	action_auth_level :index, :instructor
 	def index
 	end
+
+private
+	def get_current_metrics(course_id)
+		# returned as a hash, with each type's integer index as a key
+		# GRACE_DAY_USAGE = 1 # :grace_day_threshold, :date
+		# GRADE_DROP = 2 # :percentage_drop, :consecutive_counts
+		# NO_SUBMISSIONS = 3 # :no_submissions_threshold
+		# LOW_GRADES = 4 # :grade_threshold, :count_threshold
+		return RiskCondition.get_current_for_course(course_id)
```

Comment:
the API shouldn't be returning enums of 1, 2, 3, 4, because the user of the API might not necessarily know the exact implementation. I suggest that it be changed to a string value. 

At the same time, I don't think you can return hash You might need to convert it to json to return. 
```
class UsersController < ApplicationController
    def show
        @user = User.find(params[:id])
        render json: @user
    end
end
```
At the same time, it should return appropriate error messages as well.

Reply:
Changes pushed. Can you take another look at it? Thanks!